SETUP.md

```diff
@@ -12,7 +12,7 @@ This guide is to get you setup with the tools you'll need for development.
    3. Verify the installation: `sdk version`
 2. Install _Coretto Java 8_, [link](https://sdkman.io/usage).
    1. List the versions: `sdk list java`
-   2. Install the desired versions: `sdk install java 8.332.08.1-amzn`
+   2. Install the desired versions: `sdk install java 8.332.08.1-amzn` NOTE: This version is not available on a M1 Macbook. In that case, use: `sdk install java 8.0.333.fx-librca`
```

Comment:
Is this related to Fedora running on Apple M1?

Reply:
It is not. It is for M1 Macbook, OS 12.4.

Reply:
But this README is for:
> **Platform**: Fedora 35, 64-bit

Reply:
also, `sdk` is not a default package manager on OS X. why not use `brew`?